lib/community/bloc/community_bloc.dart

```diff
@@ -69,7 +69,7 @@ class CommunityBloc extends Bloc<CommunityEvent, CommunityState> {
     );
     on<DismissReadEvent>(
       _dismissReadEvent,
-      transformer: throttleDroppable(throttleDuration),
+      transformer: throttleDroppable(Duration.zero), // Don't give a throttle on dismiss read
```

Comment:
I kept it in to make it clear that it should not be throttled! Otherwise, I may look back at it in the future and assume that I forgot to throttle a specific request or action 😅